condacolab.py

```diff
@@ -82,19 +108,31 @@ def install_from_url(
         shutil.copyfileobj(response, out)
 
     print("📦 Installing...")
-    task = run(
+
+    condacolab_task = run_subprocess(
         ["bash", installer_fn, "-bfp", str(prefix)],
-        check=False,
-        stdout=PIPE,
-        stderr=STDOUT,
-        text=True,
-    )
-    os.unlink(installer_fn)
-    with open("condacolab_install.log", "w") as f:
-        f.write(task.stdout)
-    assert (
-        task.returncode == 0
-    ), "💥💔💥 The installation failed! Logs are available at `/content/condacolab_install.log`."
+        "condacolab_install.log",
+        )
+
+    """
+    Installing the following packages because Colab server expects these packages to be installed in order to launch a Python kernel :
+        - matplotlib-base
+        - psutil
+        - google-colab
+        - colabtools
+    """
```

Comment:
Multiline comments should use `#`. Using a triple quote string makes Python create a string object in memory only to discard it. It's not that we are slowing things a bunch, but it's not a good practice!